server/utils/stripHtml.util.js

```diff
@@ -0,0 +1,11 @@
+import stripHtml from 'string-strip-html';
+
+/**
+ * Convert html strings to plain text
+ * @param {string} string
+ * @returns {string} plain text
+ */
+
+const stripHtmlHtmlString = async string => stripHtml(string);
```

Comment:
Good job @geekdanitek , kindly explain how this function handles new line of sequence characters in the body content.

Reply:
Yeah, it will replace a new line of sequence characters with only one.